src/Symfony/Component/Validator/Resources/translations/validators.az.xlf

```diff
@@ -402,6 +402,34 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Şəbəkə maskasının dəyəri {{ min }} və {{ max }} arasında olmalıdır.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Fayl adı çox uzundur. {{ filename_max_length }} və ya daha az simvol olmalıdır.</target>
```

Comment:
Is it `Fayl adı çox uzundur.` or `Fayl adıçox uzundur.`?

Reply:
Btw. Look at the English syntax: 

```
The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.

Singular | Plural
```

Is this something you accidentally missed or is it the same phrase in Azerbaijani?

Reply:
> Is it `Fayl adı çox uzundur.` or `Fayl adıçox uzundur.`?

@Nyholm First is the correct one.

Reply:
> Btw. Look at the English syntax:
> 
> ```
> The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.
> 
> Singular | Plural
> ```
> 
> Is this something you accidentally missed or is it the same phrase (for both singular and plural) in Azerbaijani?

@Nyholm both same in Azerbaijani.